controllers/users.js

```diff
@@ -76,3 +76,54 @@ exports.modifyUser = async (req, res) => {
     },
   );
 };
+
+exports.modifyUserCreatedQuizzes = async (req, res) => {
+  await User.updateOne(
+    { _id: req.params.id },
+    { $push: { created_quizzes: req.body.created_quizzes } },
```

Comment:
may be the problem with duplicates could be solved with `arrayFilters` method? It updates a specific element of an array based on a condition (in your case the `quizId` field)